climada/util/coordinates.py

```diff
@@ -1139,13 +1139,16 @@ def get_country_code(lat, lon, gridded=False):
         region_id[region_id == -1] = 0
     return region_id
 
-def get_admin1_info(country_names):
+def get_admin1_info(countries):
```

Comment:
Changing the parameter name here ... this will break any code that names the parameter, e.g. `x = get_admin1_info(country_names='ARM')`

I've looked through core and petals and I can't find any code that does this, so it's prooobably OK? It depends on how important it is to rename the parameter, and how long this method has been part of CLIMADA.

Reply:
good point - no urgent need to change that parameter's name, so I willl leave it